### ISSUE TITLE
Content is not personalized enough

### ISSUE THREAD

**Issue by Lukeslux** [mircealungu/Unified-Multilanguage-Reader#264]
From the future work of the thesis:
- **People desire categories instead of distinct sources**.
A simple solution: create RSS feeds ourselves that aggregate a set of RSS feeds into a specific category and language. These feeds can be added to UMR without changing any code (but will still solve the entire issue). Adding feed descriptions would make this even cooler.
- **People would like to filter listed content based on attributes such as language and source**.
Somewhat more work to solve than the above problem, but doable. How to design this UI wise is the biggest problem here, since we do not want to clutter the screen with many buttons.

**Comment by mircealungu** [comment]
categories instead / alongside of sources would be great.

**Comment by Lukeslux** [comment]
Then I think the question arises: Will we implement the categories such that they are delivered by the zeeguu API, or will we implement a database in UMR and link the categories to feeds there?
The latter will require some additional care, as it won't be a simple flask blueprint anymore..

**Comment by mircealungu** [comment]
I think the place to do it is in the Zeeguu core and deliver them through the API. I wonder if we need to sit together to look at the zeeguu-core code to plan this. I don't think you looked at that code yet.

**Comment by Lukeslux** [comment]
I do not think we have gotten a proper look at that yet, indeed. We could plan a meeting where Dan and I join you when we are all back in Groningen? Trying to implement that feature would be a good practice exercise for us to learn that code then as well ;)

**Comment by mircealungu** [comment]
Sounds good. Let's do that!
